pkg/health/client/modules_test.go

```diff
@@ -30,14 +30,6 @@ func TestGetAndFormatModulesHealth(t *testing.T) {
 			h: newTestMHappy(),
 			e: "Modules Health:\tStopped(0) Degraded(1) OK(1) Unknown(0)",
 		},
-		"happy-verbose": {
-			h: newTestMHappy(),
-			e: `Modules Health:
-  Module	Status	Message	Last Updated
-  m1	OK	a ok	          2s
-  m2	Degraded	doh	         20s`,
-			v: true,
-		},
```

Comment:
Is this being removed due to https://github.com/cilium/cilium/pull/28773/files#r1372564148?

Reply:
Yes, as well as some work being done on making the CLI cleaner: https://github.com/cilium/cilium/pull/28800

However, removing this is probably not the way to go so I've re-added the test case in my last commit with some changes.